e2e_tests/cli_operations.py

```diff
@@ -23,6 +23,7 @@ def delete_deployment(deployment_type, deployment_name):
     ) as proc:
         delete_deployment_stdout = proc.stdout.read().decode('utf-8')
     logger.info(delete_deployment_stdout)
+    return delete_deployment_stdout
 
```

Comment:
This function is called here https://github.com/bentoml/BentoML/blob/master/e2e_tests/yatai_server/test_postgres_s3.py#L47 and a value is assigned to the return, however this didn't return anything